ceph.spec.in

```diff
@@ -1341,6 +1341,9 @@ cmake .. \
 %if 0%{with system_pmdk}
     -DWITH_SYSTEM_PMDK:BOOL=ON \
 %endif
+%if 0%{with jaeger}
```

Comment:
> needed for cmake optional build options during jenkins shaman builds

IMHO, this change is not directly related to jenkins or shaman. it is just an oversight in 4b69d3a1844433845eb7000f56be72fb11f4e737 which failed to enable the `WITH_JAEGER` option when `--with-jaeger`.